corehq/apps/settings/tests/test_views.py

```diff
@@ -12,6 +13,7 @@ def test_qr_code(self):
         """
         view = EnableMobilePrivilegesView()
         view.get_context_data = Mock(return_value={})
+        CouchUser.get_by_username = Mock()
```

Comment:
We offlined about this, but this is causing other tests to fail (and is not a good pattern to follow).

Reply:
Made the change